app/models/product.rb

```diff
@@ -1,11 +1,18 @@
 class Product < ApplicationRecord
   belongs_to :user
-  has_many :order_table
-  has_many :orders, through: :order_table
+  has_many :line_items
+  #has_many :orders, through: :line_items
+  belongs_to :order
 
   validates :user_id, presence: true
 
 
 
-
+  def self.search(search)
```

Comment:
## Nice 

Please use the right name for method arguments so the method will be readable

Reply:
ok